components/browser/icons/src/main/assets/mozac.browser.icons/icons-top200.json

```diff
@@ -0,0 +1,677 @@
+[
+    {
```

Comment:
I'm not quite sure how often this list is going to be updated, should be we have some automation similar to **the Public Suffix List**?

Reply:
The team said in Whistler that this happens very rarely. Also looking at the commits of the repo (42) there doesn't seem to be much activity:
https://github.com/mozilla/tippy-top-sites/commits/master

There's a hosted version of the list that I would like to switch to in the future. With that we wouldn't need to ship the list ourselves - and also not update it.

Reply:
👍